components/ws-manager-bridge/src/bridge.ts

```diff
@@ -30,15 +32,49 @@ function toBool(b: WorkspaceConditionBool | undefined): boolean | undefined {
 
 @injectable()
 export class WorkspaceManagerBridge implements Disposable {
-    @inject(TracedWorkspaceDB) protected readonly workspaceDB: DBWithTracing<WorkspaceDB>;
-    @inject(TracedUserDB) protected readonly userDB: DBWithTracing<UserDB>;
-    @inject(MessageBusIntegration) protected readonly messagebus: MessageBusIntegration;
-    @inject(PrometheusMetricsExporter) protected readonly prometheusExporter: PrometheusMetricsExporter;
+    @inject(TracedWorkspaceDB)
+    protected readonly workspaceDB: DBWithTracing<WorkspaceDB>;
+
+    @inject(TracedUserDB)
+    protected readonly userDB: DBWithTracing<UserDB>;
+
+    @inject(MessageBusIntegration)
+    protected readonly messagebus: MessageBusIntegration;
+
+    @inject(PrometheusMetricsExporter)
+    protected readonly prometheusExporter: PrometheusMetricsExporter;
+
+    @inject(Configuration)
+    protected readonly config: Configuration;
+
     protected readonly disposables: Disposable[] = [];
     protected readonly queues = new Map<string, Queue>();
 
-    public async startDatabaseUpdater(clientProvider: ClientProvider): Promise<void> {
-        const subscriber = await new WsmanSubscriber(clientProvider);
+    public start(cluster: WorkspaceCluster, clientProvider: ClientProvider) {
+        const logPayload = { name: cluster.name, url: cluster.url };
+        log.debug(`starting bridge to cluster...`, logPayload);
+
+        if (cluster.controller === this.config.installation) {
+            log.debug(`starting DB updater: ${cluster.name}`, logPayload);
+            /* no await */ this.startDatabaseUpdater(clientProvider, logPayload)
+                .catch(err => log.error("cannot run database updater", err));
```

Comment:
> what happens in this case? We have a "connected" workspace cluster whose status updates aren't persisted?

Short: Yes.
Long: IMO it make sense to leave it here for a couple of reasons:
 - we'll need the DB updater here once we plan to move (partially) away from db-sync, so it make sense to leave the logic simple here
 - it helps to see if we're able to connect
 
 > What can fail here to begin with?
 > Do we try to start again?
 
 It's not expected to fail but start itself again.
 
 > Do we fail the cluster (i.e. set its state to cordoned - probably tricky because it is exactly the database that seems to have trouble)?
 
This version (according to the design doc) contains to dynamism around `state`, e.g. we have to set that manually. And even if so, I would frown from re-using the same field but rather introduce sth ala "ClusterHealth" that holds the various aspects of what constitutes a "healthy" cluster (for instance "wsman connection drop rate", "workspaces start error rate", "workspace unknown rate", etc.).